metals/src/main/scala/scala/meta/internal/metals/debug/SourcePathAdapter.scala

```diff
@@ -0,0 +1,66 @@
+package scala.meta.internal.metals.debug
+
+import java.net.URI
+
+import scala.meta.internal.io.FileIO
+import scala.meta.internal.metals.BuildTargets
+import scala.meta.internal.metals.Directories
+import scala.meta.internal.metals.MetalsEnrichments._
+import scala.meta.io.AbsolutePath
+
+import ch.epfl.scala.bsp4j.BuildTargetIdentifier
+
+private[debug] final class SourcePathAdapter(
+    workspace: AbsolutePath,
+    sources: Set[AbsolutePath],
+    buildTargets: BuildTargets
+) {
+  private val dependencies = workspace.resolve(Directories.dependencies)
+  def toBspURI(sourcePath: AbsolutePath): Option[URI] = {
```

Comment:
```suggestion
  def toDapURI(sourcePath: AbsolutePath): Option[URI] = {
```
wouldn't make more sense?

Reply:
IDK which is better. My idea was that the paths of the source jars come from the response of the `buildTarget/dependencySources` BSP request.

Reply:
`toDapURI` would make sense in this context I think since we are in debug package.